scripts/get_latest_release.sh

```
@@ -0,0 +1,22 @@
+  else
+    latest_release=refs/tags/$latest_release
+  fi
+  if [ "$latest_release" == "" ]; then
+    echo "No release found" >&2
+    exit 1
+  fi
+  echo $latest_release
+}
+
+function main() {
+  echo $(get_latest_release)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)